website/README.md

```diff
@@ -475,9 +513,9 @@ It's also worth noting that it is possible to do glob-based redirects, for examp
 
 We support the following browsers targeting roughly the versions specified.
 
-| ![Chrome](https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_24x24.png) | ![Firefox](https://raw.githubusercontent.com/alrra/browser-logos/master/src/firefox/firefox_24x24.png) | ![Opera](https://raw.githubusercontent.com/alrra/browser-logos/master/src/opera/opera_24x24.png) | ![Safari](https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari/safari_24x24.png) | ![Internet Explorer](https://raw.githubusercontent.com/alrra/browser-logos/master/src/edge/edge_24x24.png) |
```

Comment:
Need to verify if we want these changes.

Reply:
Hot take: these changes look good to me, since I think we explicitly stopped supporting IE11 at some point a while back?